erpnext/projects/doctype/project/project.py

```diff
@@ -412,11 +412,29 @@ def create_project_update(project):
 			)
 	return data
 
+def update_project_daily():
+	if frappe.db.get_single_value("Selling Settings", "sales_update_frequency") != "Daily":
+		return
+	for project in frappe.get_all('Project'):
+		project = frappe.get_doc('Project', project.name)
+		project.update_sales_amount()
+		project.update_billed_amount()
+		project.save()
+
+def update_project_monthly():
+	if frappe.db.get_single_value("Selling Settings", "sales_update_frequency") != "Monthly":
+		return
+	for project in frappe.get_all('Project'):
```

Comment:
this is same as the earlier function, please make a common function for the project update